docs/contributing/testing-guidelines/integration-testing.md

```diff
@@ -0,0 +1,266 @@
+# Integration testing
+
+## Introduction
+
+The purpose of this guide is to motivate developers to adopt integration testing best practices with Autoware by explaining how to write, run,
+and evaluate the results of integration tests.
```

Comment:
I think we can drop the section name here.
@LalithVipulananthan I remember you mentioned this kind of rule before. What do you think?

```suggestion
The purpose of this guide is to motivate developers to adopt integration testing best practices with Autoware by explaining how to write, run,
and evaluate the results of integration tests.
```

Reply:
I agree. Done in 4b5d4b9e24ecf560e0b33d4c328b3c42dca76859